modules/imgproc/src/lsd.cpp

```diff
@@ -156,6 +162,12 @@ inline double log_gamma_lanczos(const double& x)
     }
     return a + log(b);
 }
+
+// Compare norm
+inline bool compare_norm(const coorlist& n1, const coorlist& n2)
```

Comment:
You can put this in class `LineSegmentDetectorImpl` as **static** member function.

Reply:
I added this function in LineSegmentDetectorImpl class as static member function of public member.